### PR TITLE
Noise distortion test stage

### PR DESCRIPTION
**Pull request by Mati-AAU**

### CODE REVIEW

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import dk.scuffed.opengltest.R
+import dk.scuffed.opengltest.gl.*
+import dk.scuffed.opengltest.pipeline.*
+import java.nio.ByteBuffer
+
+
+internal class NoiseDistortionStage(private val context: Context, private val inputFrameBufferInfo: FramebufferInfo, private val pipeline: Pipeline): GLOutputStage(context, R.raw.vertex_shader, R.raw.shaderfun, pipeline) {
+
+    // Texture data
+    lateinit var textureBuffer : ByteBuffer
```
Comment (mads256h) [inline app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: private

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    private lateinit var textureUnitPair: TextureUnitPair
+
+    private var time = System.currentTimeMillis();
+
+    var textureHandle : Int = 0
```
Comment (mads256h) [inline app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: private

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import java.nio.ByteBuffer
+
+
+internal class NoiseDistortionStage(private val context: Context, private val inputFrameBufferInfo: FramebufferInfo, private val pipeline: Pipeline): GLOutputStage(context, R.raw.vertex_shader, R.raw.shaderfun, pipeline) {
+
+    // Texture data
+    lateinit var textureBuffer : ByteBuffer
+    private lateinit var textureUnitPair: TextureUnitPair
+
+    private var time = System.currentTimeMillis();
```
Comment (mads256h) [inline app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: val

**Review by mads256h — CHANGES_REQUESTED** [review]
smol changes :)

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt (hunk 3 above)
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: done

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt (hunk 2 above)
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: done

File: app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt (hunk 1 above)
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/opengltest/pipeline/stages/NoiseDistortionStage.kt]: done

**Review by mads256h — APPROVED** [review]

**Pull request merged**